evadb/executor/create_database_executor.py

```diff
@@ -26,6 +28,15 @@
 class CreateDatabaseExecutor(AbstractExecutor):
     def __init__(self, db: EvaDBDatabase, node: CreateDatabaseStatement):
         super().__init__(db, node)
+        if node.engine == "slack":
+            mod = importlib.import_module(
+                "evadb.third_party.applications.slack.slack_chatbot"
+            )
+            self._slack_chatbot = mod.SlackChatbot()
+
+    @property
+    def slack_chatbot(self):
+        return self._slack_chatbot
```

Comment:
Database specific code in the executor is generally not a good idea. It will be great we can think how to modularize the implementation.

Reply:
But where could we create the chatbot instead?

Reply:
We only use `self.slack_chatbot` to trigger running the chatbot.

Reply:
Hi Kaushik, based on our discussion earlier today, since this will not be integrated as a handler, can we just start the chatbot as a standalone file as an external application? Then, it can call whatever query is needed through EvaDB. You can check other examples in the `app` folder for reference.

Reply:
I have a similar understanding with Jiashen. The handler is for support `SELECT * FROM slack_data` and in the future `INSERT`. The slack chat bot in this PR is likely to be a usecase for EvaDB. For example, the EvaDB stores the embedding and index of pdf files. And when the slack chat bot received the questions, it will query the EvaDB to get an answer and returns to the user.   If that's the case, the slackbot will be more like the youtube_qa in https://github.com/georgia-tech-db/evadb/tree/staging/apps

Reply:
With further discussion today, we decided on a more extensive plan for this. Closing this PR for now. Will reopen a new one